ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -140,6 +127,17 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
+    <Reference Include="Ionic.Zip">
```

Comment:
Thanks @museyoucoulduse, can you please manually move these lines back within notepad? Then we can merge this in.